### ISSUE TITLE
Z-index incompatibility with the most recent version of Laravel Nova

### ISSUE THREAD

**Issue by antennaio** [whitecube/nova-flexible-content#488]
First things first, thanks for this wonderful package!

I would like to report a z-index related incompatibility with the latest version of Laravel Nova (v4.29.5).

A screenshot explains it best:

![image](https://github.com/whitecube/nova-flexible-content/assets/530421/e5d677f4-4414-4762-9712-9bd49b759d60)

Release notes for [Laravel Nova v4.29.5](https://nova.laravel.com/releases/4.29.5) mention _Fixed some more issues with z-index stacking._

**Comment by ralphschindler** [comment]
I have this problem as well.

**Comment by Senexis** [comment]
Encountered this is as well.

Potentially useful to add: Instead of having to do with `z-index`, it seems to have to do with the addition of `.overflow-hidden` on cards.

**Comment by toonvandenbos** [comment]
Hello everyone,

Please report this issue to Nova's team as this will probably impact many other packages as well. If they do not want to fix the `overflow-hidden` issue on their part we'll have to position the dropdown differently.

**Comment by wize-wiz** [comment]
@tjmckenzie Seems to be the same fix as https://github.com/outl1ne/nova-menu-builder/pull/186/files.

I just don't seem to replicate this problem in 4.31.3, though the [pull](https://github.com/outl1ne/nova-menu-builder/pull/186) was created 2 weeks ago so somehow it seems to cause problems.

**Comment by alekmlynek** [comment]
I still have this issue. I'm going to attempt the CSS fix above, but it feels hacky.

**Comment by mstaack** [comment]
the css mentioned above works with the latest nova version. @alekmlynek you probably have cached content

**Comment by DrNajim** [comment]
For the latest verion of laravel nova
        "laravel/nova": "^4.32",

I just solved the issues by adding this at NovaServiceProvider
       Nova::style('custom', asset('css/styles/nova.css')); 
       
I added this at public/css/styles/nova.css
div.relative.overflow-hidden:has(div > div > label[for$="nova-flexible-content"]) {
    overflow: visible !important;
}
or this 
div.relative.overflow-hidden:has(div > div > label[for$="content-creer-page-nova-flexible-content"]) {
    overflow: visible !important;
}

**Comment by siljva** [comment]
I'm having this same issue
![image](https://github.com/whitecube/nova-flexible-content/assets/100735858/8a063be1-2461-4135-9587-9e5d3b1c4c45)

I think that css "hacks" to remove oveflow hidden are wrong. It's better to open the dropdown upwards or have it done using VueJS teleport.

https://vuejs.org/guide/built-ins/teleport.html

If I catch some spare time I'll try to fix it and make a PR.

**Comment by harmenjanssen** [comment]
I agree with @siljva: opening upwards is probably the easiest fix. 
The above posted CSS for removing `overflow: hidden` is not working for me.

**Comment by alekmlynek** [comment]
@DrNajim @mstaack You're right, I wasn't paying attention to the code snippet. It is important to edit the label content with the actual field itself, which for me wasn't "nova-flexible-field".

For anyone else deploying this:
```
div.relative.overflow-hidden:has(div > div > label[for$="CHANGE-THIS-TO-YOUR-FIELD-LABEL-IF-NOT-DEFAULT"]) {
    overflow: visible;
}
```

**Comment by damarinov1** [comment]
So, not going to be fixed and we need to override it with our CSS? Such a nova way of doing stuff 🙄

**Comment by chrispage1** [comment]
@damarinov1 make the move to Filament, it's what nova should have been 😅

**Comment by mstaack** [comment]
my colleague gave me this working css:
```
.modal > div:first-child > form {
    overflow: visible !important;
}
```
